src/Props/sponsors/partners/props.ts

```diff
@@ -0,0 +1,69 @@
+import axure from "images/sponsors/partners/axure.png"
+import balsamiq from "images/sponsors/partners/balsamiq.png"
+import earthhacks from "images/sponsors/partners/earthhacks.png"
+import echo3D from "images/sponsors/partners/echo-3D.png"
+import googleCloud from "images/sponsors/partners/google-cloud.png"
+import mlh from "images/sponsors/partners/mlh.png"
+import stickermule from "images/sponsors/partners/sticker-mule.png"
+import wolfram from "images/sponsors/partners/wolfram.png"
+import woodstocks from "images/sponsors/partners/woodstocks.png"
+
+export interface PartnerProps {
+  image: string
+  url: string
+  key: string
+}
+
+export const row1: PartnerProps[] = [
+  {
+    image: mlh,
+    url: "",
```

Comment:
Did you mean to put URL's here before we release to production.

Reply:
That's a good idea to have clickable links that redirect properly so people aren't confused. I've updated it temporarily with divs mainly because we still haven't received the sponsor links yet but our timeline really needs this merged in for additional funding so this might have to be another chore added in the future.